app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    pass

```

Comment:
The `NotWearingMaskError` class should include a descriptive error message. Consider adding a constructor to accept a message parameter or provide a default message.